src/main/target/KISSFCV2F7/target.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+#define TARGET_BOARD_IDENTIFIER "KISSV2"
+
+#define USBD_PRODUCT_STRING "KISSFCV2F7"
+
+#define SBUS_PORT_OPTIONS (SERIAL_STOPBITS_2 | SERIAL_PARITY_EVEN | SERIAL_INVERTED | SERIAL_BIDIR)
```

Comment:
This should not be done. `SERIAL_STOPBITS_2 | SERIAL_PARITY_EVEN` are defaults anyway, adding `SERIAL_INVERTED` and `SERIAL_BIDIR` to `SBUS_PORT_OPTIONS` will make it impossible for users to switch these off - in an unintuitive way.